mix.exs

```diff
@@ -74,47 +74,48 @@ defmodule DotCom.Mixfile do
       {:briefly, "~> 0.3"},
       {:bypass, "~> 1.0", [only: :test]},
       {:castore, "~> 0.1.11"},
-      {:con_cache, "~> 0.12.0"},
+      {:con_cache, "~> 1.0.0"},
```

Comment:
I thought the plan was to pin dependencies to a specific version. Could just take the version from the lock file and put it in here.

Reply:
We never decided on a solution. Just commented on the semvar behavior. I can lock these dependencies to one version if that is desired.

Reply:
Yeah, we should. To wit, `~> 1.0.0` won't upgrade to `1.1.0`.